packages/client/src/components/app/blocks/RowExplorer.svelte

```diff
@@ -0,0 +1,247 @@
+<script>
+  import Block from "components/Block.svelte"
+  import BlockComponent from "components/BlockComponent.svelte"
+  import { makePropSafe as safe } from "@budibase/string-templates"
+  import { generate } from "shortid"
+
+  export let dataSource
+  export let height
+
+  export let cardTitle
+  export let cardSubtitle
+  export let cardDescription
+  export let cardImageURL
+  export let cardSearchField
+
+  export let detailFields
+  export let detailTitle
+
+  const stateKey = generate()
+
+  let listDataProviderId
+  let listRepeaterId
+</script>
+
+<Block>
+  <BlockComponent
+    type="container"
+    props={{
+      direction: "row",
+    }}
+    styles={{
+      custom: `
+        height: ${height} !important;
```

Comment:
I was wondering why height was implemented as a setting rather than a style, but then realised it's because you'd have to pass the style down to the correct component and the other blocks don't actually do this either. Let's leave this as is for now, but I'll do an update at some stage to make blocks actually respect styles applied to them.